### PR TITLE
Update snap binary names and paths

### PR DESCRIPTION
**Pull request by mkleina**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,17 +70,17 @@ Other paths to files should be set according to your configuration, using a file
 
 In one terminal window, open the snap daemon (in this case with logging set to 1,  trust disabled and global configuration saved in config.json ):
 ```
-$ $SNAP_PATH/bin/snapd -l 1 -t 0 --config config.json
+$ $SNAP_PATH/bin/snapteld -l 1 -t 0 --config config.json
```
Comment (katarzyna-z) [inline README.md]: Could you correct $SNAP_PATH in this documentation?

See this [example](https://github.com/intelsdi-x/snap-plugin-collector-cpu#examples)

If you correct it, please change commit message and tile to of pull request to have full information in them.

**Review by katarzyna-z — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mkleina) [reply README.md]: Ok, SNAP_PATH updated

Comment (katarzyna-z) [reply README.md]: Now path should be without ''bin", so it should be
```
 $ $SNAP_PATH/snapteld -l 1 -t 0 --config config.json
```
In my environment I have binary files in this localization
```
 /home/kzab/work/src/github.com/intelsdi-x/snap/build/linux/x86_64
```

Comment (mkleina) [reply README.md]: Ok, binary path changed, now it is $SNAP_PATH instead of $SNAP_PATH/bin

**Review by katarzyna-z — APPROVED** [review]
LGTM

**Pull request merged**